app/models/miq_ae_method.rb

```diff
@@ -30,6 +30,22 @@ class MiqAeMethod < ApplicationRecord
   AVAILABLE_SCOPES     = ["class", "instance"]
   validates_inclusion_of  :scope,     :in => AVAILABLE_SCOPES
 
+  scope :name_path_search, lambda { |search|
+    where('name ILIKE ? or relative_path ILIKE ?', "%#{search}%", "%#{search}%") if search.present?
+  }
+
+  scope :domain_search, lambda { |domain_id|
+    where(:domain_id => domain_id) if domain_id.present?
+  }
+
+  scope :selected_methods, lambda { |method_ids|
+    if method_ids.present?
+      where(:id => method_ids.split(',').map(&:to_i))
+    else
+      where(nil)
```

Comment:
unsure, but think a simple `self` would work here.

Reply:
Hey @kbrock , Yes, `self` methods would work here.

I used `scope` since the query was not that complex.

Will change to `self` so that it would look similar to other methods.

Reply:
then lets use `where(nil)`